app/views/IsThisYourBusinessView.scala.html

```diff
@@ -36,12 +36,12 @@ <h1 class="govuk-heading-xl">@messages("isThisYourBusiness.heading")</h1>
 
         <p class="govuk-body">@businessName</p>
 
-        <p class="govuk-body">@address</p>
+        <p class="govuk-body">@Html(address)</p>
```

Comment:
It is not a good idea to pass a string. You have an Address object and this is the one that needs to be passed to the template instead of string

Reply:
Changed.